#######################################################################
# This message was automatically generated by the ark_bridge package
#######################################################################
#
# Source message:    .msg
# Creation:          Thu 13 Apr 2017 12:02:49 PM EDT
#
#######################################################################
Header              header
float32             angle_min
float32             angle_max
float32             angle_increment
float32             time_increment
float32             scan_time
float32             range_min
float32             range_max
int32               nranges
int32               nintensities
float32[]           ranges
float32[]           intensities
